jest.config.ts

```diff
@@ -7,6 +7,7 @@ const customJestConfig: Config.InitialOptions = {
   },
   moduleDirectories: ['node_modules', '<rootDir>/'],
   setupFilesAfterEnv: ['<rootDir>/setupTests.ts'],
+  testMatch: ['<rootDir>/src/__tests__/**/*.spec.ts'],
```

Comment:
I changed this so that I could be the msw folder in the __tests__ folder without it testing the handlers.ts file for example